src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -121,11 +116,12 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
 
       <div className="room-modal__main">
         <div className="room-modal__icon" style={iconStyles} />
-
-        <RoomModalOngoingEvent
-          roomEvents={roomEvents}
-          onRoomEnter={enterRoomWithSound}
-        />
+        {venueEvents && (
+          <RoomModalOngoingEvent
+            roomEvents={venueEvents}
+            onRoomEnter={enterRoomWithSound}
+          />
+        )}
```

Comment:
This previously wasn't needed because we explicitly ensured that there would always be a defined array here. That changed based on @mike-lvov's comments.

Instead of conditionally rendering here, we should continue to allow `RoomModalOngoingEvent` to make its own decisions. It's written to handle an array, even when it's empty, so we should give it that.

I don't really know why @mike-lvov suggested to make the param optional in both of these components, and IMO it's a regression and not something we should be doing. By switching it back to always being defined, we can leave this as it was, which worked well.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/171b2904631fd39b954ae9727b9b4ef0be5af6b0